ui/js/controllers/main.js

```diff
@@ -212,7 +212,9 @@ treeherderApp.controller('MainCtrl', [
          */
 
         $scope.isSingleTierSelected = function () {
-            return _.without(_.values($scope.tiers), false).length === 1;
+            return Object.values($scope.tiers).filter(function (value) {
```

Comment:
let's use the more terse ES6 syntax like:

    return Object.values($scope.tiers).filter(value => (value !== false)).length === 1;